website/content/project/presentation.md

```diff
@@ -26,50 +34,71 @@ You must also discuss at least one thing that you would've done differently with
+2. Allocate 15 minutes to deliver your presentation, and ask students in other teams whether they would be willing to attend the meeting. This will make your presentation less awkward, because you'll actually be presenting to other people, and you will feel more obligated to stay within the 15 minute limit (as opposed to getting into a "let's do a retake" mentality). Plus, the people from the other teams may give you some valuable feedback at the end of the meeting (and will themselves benefit from seeing how another team approaches giving a presentation; you should also volunteer to attend other presentations precisely for this reason!)
+3. If feasible, deliver the presentation again at a later time, incorporating the feedback you received from the other students.
 
-## Scoring
+All that said, you are allowed to record your presentation in other ways if you feel that will work better for you. However, if you've never prepared a pre-recorded presentation before, we suggest you use Zoom as described above.
+
+We suggest you allocate roughly 5 minutes for each portion of the presentation. Please note that the design/implementation and demo portions can be interwoven together, as opposed to treating them as completely distinct portions.
+
+Take into account that the presentation itself doesn't have to be delivered by all team members. In fact, for a 15 minute presentation, you probably want no more than 2-3 people speaking in total. If you feel comfortable doing so, please make sure your camera is turned on when you speak during the presentation. 
+
+## Submission Instructions
 
-The presentation will be worth 10 points of your final grade. 6 of these points will be determined by the instructor and TAs, and the remaining 4 points will be determined by a panel of guest judges. The  instructor/TAs will score your presentation (on a scale of 0 to 5) on three aspects: Clarity/structure (weighed 40%), demo (weighed 35%) and Q&A (weighed 25%). The guest judges will score your presentation only on clarity/structure (75%) and Q&A (25%).
+Submission instructions will be posted closer to the deadline, but we expect you will be able to upload your video through Panopto.
 
-Both the instructor/TAs and guest judges will use the following rubric, with discretion to add/subtract half a point.
+## Q&A
 
-### Clarity and structure
+During the Q&A meeting, you will be asked some questions based on your presentation (this is meant to emulate the Q&A that typically happens after a live presentation). As many team members as possible should attend the Q&A meeting, sp
```

Comment:
I think the end of this section got deleted

Reply:
Oops, fixed.